mod-orders-storage/schemas/cost.json

```diff
@@ -3,14 +3,35 @@
   "description": "purchase order line cost record",
   "type": "object",
   "properties": {
-    "listPrice": {
-      "description": "the per-item list price",
+    "listUnitPrice": {
+      "description": "The per-item list price",
+      "type": "number"
+    },
+    "listUnitPriceElectronic": {
+      "description": "The e-resource per-item list price",
       "type": "number"
     },
     "currency": {
-      "description": "a currency code",
+      "description": "An ISO currency code",
       "type": "string"
     },
+    "additionalCost": {
+      "description": "Lump sum that is added to the total estimated cost - not affected by discount",
+      "type": "number"
+    },
+    "discount": {
+      "description": "Percentage or amount that is subtracted from the list price time quantities calculation before additional cost",
```

Comment:
Nice description, it would be nice to add the same level of detail to the totalEstimated price description too

Reply:
We might want to note that if a discountType = Percent,  the value for 10 percent would be expressed as 0 not 0.1

Reply:
The description is updated highlighting allowed values:
`Percentage (0 to 100) or amount (positive number) that is subtracted from the list price time quantities calculation before additional cost`